FreeRTOS/FreeRTOS
# Latest Percepio Tracealyzer Recorder v4.3.11

Latest Percepio Tracealyzer Recorder, v4.3.11

Description
-----------
The recorder in the repository is greatly outdated and doesn't compile with later FreeRTOS versions.
As part of Percepio's new release procedure we will be creating pull requests to keep the trace recorder in FreeRTOS-Plus up-to-date.

Test Steps
-----------
N/A

Related Issue
-----------
N/A


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice. Note that the terms of usage stated by Percepio is provided in the source files.
